models/endoscopic_tool_segmentation/docs/README.md

```diff
@@ -100,6 +100,12 @@ For more details usage instructions, visit the [MONAI Bundle Configuration Page]
 python -m monai.bundle run --config_file configs/train.json
 ```
 
+The default dataset path is defined as "/workspace/data/endoscopic_tool_dataset" in "configs/train.json", please modify it to the actual path via modifying `dataset_dir`:
+
+```
+python -m monai.bundle run --config_file configs/train.json --dataset_dir <actual dataset path>
+```
+
```

Comment:
Hi @zephyrie , as for the `dataset_dir` notification, I added this part, could you please help to review it? If it's fine, I will adapt to all bundles.

Reply:
cc @Nic-Ma

Reply:
@yiheng-wang-nv, that looks good!

Reply:
Thanks @zephyrie , I did a bit change to the sentence, in order to match up with some bundles that are already mention the `dataset_dir` in readme.